src/utils/index.ts

```diff
@@ -0,0 +1,4 @@
+// allow consuming libraries to provide their own Promise implementations
+export default {
+  Promise: window && window['Promise']
+};
```

Comment:
i'm not gonna pretend i understand how this works.

Reply:
It probably doesn't.

Reply:
I'm working on pulling this into ember-esri-loader right now, so I'll find out real soon.

Reply:
it works!